app/models/school_data_importer.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "file_download"
+require "csv"
+
+class SchoolDataImporter
+  def run
```

Comment:
We need to think about how we handle schools that close

Reply:
Well, the good part is that GIAS data tells us when school closes.

I imagine the approach, when we decide what to do, is to have an `after_update` hook on a school model, that will do _something_. That will probably be enough for school closing.

For school change, uhh, most of them will just be saved in the db, thanks to urn matching. For schools merging, or splitting, argh, I don't even want to think about it. Not sure how it gets represented in gias, and I agree we will need to dig around that. I can add a TODO here, maybe with a ticket number describing the problem?

Reply:
I think that would be good for now 👍